demo/simple/scroll.html

```diff
@@ -0,0 +1,50 @@
+<head>
+  <title>Virtual Scoller Demo - Scroll</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <style>
+    .scroller {
+        overflow: scroll;
+        height: 400px;
+    }
+  </style>
+</head>
+
+<body>
+  <p>Scroll a little every frame. This will be slower as the virtual
```

Comment:
"but hopefully not by much", maybe?

Reply:
Add "but hopefully fast enough to avoid dropping a frame."